graphics/asymptote/Portfile

```diff
@@ -21,32 +22,40 @@ license             LGPL-3+
 # libpaper is a dependency of ghostscript that's only used at runtime
 license_noconflict  libpaper
 
-set python.branch   3.9
+set python.branch   3.10
```

Comment:
this should use Python 3.11 if possible

Reply:
I will try with 3.11, and thank you for reminding about linking, I will look into it.